inner_code/filter_functions.py

```diff
@@ -0,0 +1,63 @@
+from typing import Tuple, Union
+def filter_gc_content(gc_bounds: Union [tuple, int], key: str, value: Union [tuple, int]):
+    """
+    Filters sequences asccording to GC-content\n
+    Arguments:\n
+    gc_bounds (tuple, int) - desired boundaries for GC-content. Default boundaries - (0, 100)\n
+    key - key from main function dictionary\n
+    value - value from main function dictionary\n
+    Return:\n
+    bad_keys - a list with inappropriate sequences
+    """  
+    bad_keys1 = []
+    if len(gc_bounds) == 1:
+        gc_bounds.append(0)
+        gc_bounds.sort()
+    gc_count = 0
+    for i in value[0]:
+        if(i == 'G' or i == 'C'):  
+            gc_count += 1
+    gc_content = round(((gc_count/len(value[0])) * 100), 2)

```

Comment:
```suggestion
    gc_content = round(((gc_count / len(value[0])) * 100), 2)

```
Тут стоит обратить внимание на PEP8